manifests/resource/location.pp

```diff
@@ -358,7 +358,7 @@
     $content_real = template('nginx/vhost/locations/empty.erb')
   }
 
-  if $ensure == present and $fastcgi != undef and !defined(File[$fastcgi_params]) {
+  if $ensure == present and $fastcgi != undef and ! empty($fastcgi_params) and !defined(File[$fastcgi_params]) {
```

Comment:
spacing could be more consistent here, probably should be `!empty` to match the rest of the line.